src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/domain/entity/update/AuditUpdate.kt

```diff
@@ -0,0 +1,15 @@
+package uk.gov.justice.digital.hmpps.hmppsaccreditedprogrammesapi.domain.entity.update
```

Comment:
Why would we update audit? Don't think we ever would/should

Reply:
Absolutely, audit must be write-only